modules/ROOT/pages/performance/spark.adoc

```diff
@@ -215,8 +215,13 @@ While for node and relationship extraction, you leverage the https://neo4j.com/d
 about the nodes/relationships you're trying pulling off, for the (3) you have two possible approaches:
 
 * Compute a count over the query that you're using.
-* Compute a count over a second *optimized* query that leverages indexes. In this case, you can pass
-it via the `.option("query.count", "<your cypher query>")` the query must always return only
+* Compute a global count over a second *optimized* query that leverages indexes. In this case, you can pass
```

Comment:
I added "global" to emphasize that the value to set is not a count per partition, but a count later divided by the number of partitions.
There likely is a better to highlight this.

Reply:
I think it's fine as a "quick patch". The whole section would benefit from a thorough review.